### ISSUE TITLE
`api.toJSON()` and `api.fromJSON()` should be be renamed, or updated to handle JSON strings.

### ISSUE THREAD

**Issue by brochington** [mathuo/dockview#619]
Currently the `api.toJSON()` and `api.fromJSON()` methods act upon deserialized object representations of the layout (JS Objects). This can lead to some confusion, as it's not technically JSON that they are using. 

There is nothing wrong with the api methods taking JS objects, but I do feel like they should be renamed to reflect what is actually happening, perhaps something like `api.saveLayout()` and `api.loadLayout()`.

**Comment by e7acampos** [comment]
I think they are perfectly named. What you are calling a JS object is actually called a Json Object.

**Comment by BeepBot99** [comment]
> I think they are perfectly named. What you are calling a JS object is actually called a Json Object.

No. JSON is a data serialization format where data is stored as a string. JavaScript objects are stored on the heap. JSON can be converted to a JavaScript object using `JSON.parse()` and objects can be converted to JSON using `JSON.stringify()`.
